create-services-plugin.go

```diff
@@ -138,7 +138,33 @@ func (c *CreateServicePush) createService(name, broker, plan, JSONParam string)
 		result = c.run("create-service", broker, plan, name, "-c", JSONParam)
 	}
 
-	return result
+	if result != nil {
+		return result
+	}
+
+	pb := NewProgressSpinner(os.Stdout)
+	for {
```

Comment:
15 minutes is a long time, but we have services (Amazon RDS) that can take well over 30 minutes to provision.  I hesitate to add a timeout here because, well, asynchronous service brokers take as long as they need to provision services and it seems unlikely that we could pick an arbitrary timeout value that would cover all use cases.  A well-behaved broker will be sending status messages back to keep the client informed of what it's doing (and of course if the broker fails the loop will break).  Worse case scenario, the Cloud Controller itself will hit its timeout and break the loop, but I think the default CC timeout is something absurd, like one week.  :)